app/services/mapping/governmentagencygoodsitem/PackagingBuilder.scala

```diff
@@ -25,40 +28,70 @@ import wco.datamodel.wco.declaration_ds.dms._2.{
 
 import scala.collection.JavaConverters._
 object PackagingBuilder {
+  def createPackaging(packageInfo: PackageInformation, index: Int): governmentagencygoodsitem.Packaging =
+    governmentagencygoodsitem.Packaging(
+      sequenceNumeric = Some(index),
+      typeCode = packageInfo.typesOfPackages,
+      quantity = packageInfo.numberOfPackages,
+      marksNumbersId = packageInfo.shippingMarks
+    )
+
+  def buildThenAdd(
+    exportItem: ExportItem,
+    wcoGovernmentAgencyGoodsItem: GoodsShipment.GovernmentAgencyGoodsItem
+  ): Unit =
+    exportItem.packageInformation.zipWithIndex.foreach { packingWithIndex =>
```

Comment:
To make it more readable you can change `packingWithIndex` to `case (packing, index) => `

it'll be more readable and easier to use

Reply:
Yeh thats a nice little trick for whenever we have `(x,y,z....)` pairs/triples etce

Reply:
ok will change, agree this is niceer